component-classes/index.js

```diff
@@ -53,7 +53,7 @@ export const box = {
   box: 'group block relative break-words last-child:mb-0 p-16 rounded-8', // Relative here enables w-clickable
   bleed: '-mx-16 sm:mx-0 rounded-l-0 rounded-r-0 sm:rounded-8', // We target L and R to override the default rounded-8
   info: 's-bg-info-subtle',
-  neutral: 's-bg-subtle',
+  neutral: 's-bg-sunken',
```

Comment:
I tried testing these changes in React by linking them there + copying over missing tokens, but the `--w-s-color-background-sunken` token that this class was looking for was missing.
<img width="378" alt="Screenshot 2024-03-07 at 10 42 58" src="https://github.com/warp-ds/css/assets/41303231/da9ca224-ec49-4b26-9ef6-5d9a8fcbcb21">